uwblueprint/community-fridge-kw
# fix: Add error handling for loading users for admin

## Brief description. What is this change? 
### [Error in User Management table](https://www.notion.so/uwblueprintexecs/Error-in-User-Management-Table-cc85cd1e6649460faed8af2d931a3069)


## Implementation description. How did you make this change?
* Added error handling in User Management table by catching error and displaying message



## Steps to test
1. Force/fake an error by changing the route in `DonorAPIClient` function `getAllDonors` to `/donor` instead of `/donors`
2. This will cause an error in loading user data
3. Log in as admin and go to User Management page
4. There should be an error message
5. Remove the fake error by changing it back to `/donors`
6. Go back to User Mgmt table and it should work as expected.


## Checklist
